ci/taos/webapp/install-nnstreamer-packages.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+##
+# @file install-nnstreamer-packages.sh
+# @brief This script is to install packages that are required to run nnstreamer modules.
+#
+# We assume that you use Ubuntu 16.04 x86_64 distribution.
+#  If you do not use Version 16.04, you have to modify some files appropriately. 
+
+
+echo -e "########## Installing base packages for Gstreamer"
+sudo apt-get -y install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libglib2.0-dev
+sudo apt-get -y install gstreamer1.0-tools gstreamer1.0-plugins-base gstreamer1.0-plugins-good
+
+echo -e "########## Installing base packages to test nnstreamer apps"
+sudo apt-get -y install rpmlint ctags sudo
+sudo apt-get -y install cmake make
+
+sudo apt-get -y install python-gi python3-gi
+sudo apt-get -y install python-gst-1.0 python3-gst-1.0
+sudo apt-get -y install python-gst-1.0-dbg python3-gst-1.0-dbg
+
+# echo -e "########## Setting-up tensorflow build environment"
+# if [[ ! -f /etc/apt/sources.list.d/taos.list ]]; then
```

Comment:
We need to make a PPA at the launchpad.net to install the tensorflow-dev and tensorflow-lite-dev package for nnstreamer.

Reply:
@again4you  PTAL.